BREAKING_CHANGES.md

```diff
@@ -111,6 +111,24 @@ should be re-checked for ID naming accuracy.
   - previously `<ThemeProvider document={doc}>`
   - currently `<ThemeProvider theme={{ ...DEFAULT_THEME, document: doc }}>`
 
+## @zendeskgarden/react-tooltips
+
+- Removed exports
+  - LightTooltip
+  - TooltipView
+  - GARDEN_PLACEMENTS
+  - POPPER_PLACEMENTS
+  - getPopperPlacement
+  - getRtlPopperPlacement
+- rename `arrow` prop to `showArrow`
```

Comment:
We should stick with our boolean naming for these changes, so `hasArrow`.

Reply:
Updated.